### DISCUSSION TITLE
LazyQuery Equivalent

### DISCUSSION THREAD

**Discussion by lmuller18** [TanStack/query#982] [General]
I was wondering if there was an equivalent to the `useLazyQuery()` that Apollo Hooks offers. I know that they are completely separate projects, but I do enjoy being able do `const [doQuery, queryResults] = useLasyQuery(...)` and would love to do something similar in React Query. Is the normal approach to connect a function that toggles the `enabled` boolean?

**Comment by jonathanstanley** [comment]
see https://github.com/tannerlinsley/react-query/discussions/1549
